api/client/client.go

```diff
@@ -1636,7 +1636,9 @@ func (c *Client) GetMFADevices(ctx context.Context, in *proto.GetMFADevicesReque
 	return resp, nil
 }
 
+// Deprecated: Use GenerateUserCerts instead.
```

Comment:
TODO: Remove in v16?

We do guarantee compatibility only +/- one version. I don't think that API is an exception.

Reply:
I usually only leave one marker per cleanup and chase the rest when I do it, otherwise I have to leave "DELETE IN" everywhere.